sim/druid/frenzied_regeneration.go

```diff
@@ -7,7 +7,7 @@ import (
 )
 
 func (druid *Druid) registerFrenziedRegenerationCD() {
-	actionID := core.ActionID{SpellID: 22842}
+	actionID := core.ActionID{SpellID: 22896}
```

Comment:
Can we also do this talent by level? There seem to be 3 ranks for 10/15/20 rage accordingly
https://www.wowhead.com/classic/spell=22842/frenzied-regeneration
https://www.wowhead.com/classic/spell=22895/frenzied-regeneration
https://www.wowhead.com/classic/spell=22896/frenzied-regeneration